src/include/runtime_config.h

```diff
@@ -35,11 +35,12 @@
 #define __DEFAULT_FAIL_MESSAGE__ CFG_STR(12)
 #define __GLOBAL_INCLUDE_FILE__ CFG_STR(13)
 #define __MUD_IP__ CFG_STR(14)
+#define __PORT_CONFIG__ CFG_STR(15)
 
 /*
  * These config settings return an int (ie number)
  */
-#define BASE_CONFIG_INT (BASE_CONFIG_STR + 15)
+#define BASE_CONFIG_INT (BASE_CONFIG_STR + 100)
```

Comment:
I would say, when somebody uses `CFG_STR(20)` and expects to get `__COMPILER_STACK_SIZE__ `, then it´s his own problem...
Alas, this is the very reason for this include file...
I know:
- which values are available
- which value results in what return type
- what the meaning is behind which value
- and (most important) which name I can use to get the desired result, regardless of driver changes